src/Calendar/Calendar.js

```diff
@@ -350,7 +391,7 @@ class Calendar extends Component {
         for (let index = 0; index < 7; index++) {
             weekDays.push(
                 <th className='fd-calendar__column-header' key={index}>
-                    <span className='fd-calendar__day-of-week'>
+                    <span className='fd-calendar__day-of-week' role='button'>
```

Comment:
Why do the days of the week have a role of button?  They aren't actually clickable, are they?